### DISCUSSION TITLE
Problem generating notices 5 and 13

### DISCUSSION THREAD

**Discussion by luispozoWS** [OP-TED/eForms-SDK#814] [Q&A] [answered]
Hello,

We have a problem generating notices 5 and 13:
« displaytype » is « SECTION » and not « GROUP ». It causes a problem when generating the notices.
How to deal with this ?

![F13](https://github.com/OP-TED/eForms-SDK/assets/158502029/f0496bd5-5dd1-4d84-8014-adbdc715c984)
![F05](https://github.com/OP-TED/eForms-SDK/assets/158502029/a75dd69c-97e5-4224-b595-ba9eef384f8b)

**Comment by rouschr** [comment]
Hello,

Are you using enotices2 or your own application? For enotices2 normally you should use the enotices2 helpdesk ( https://enotices2.ted.europa.eu/help/contact)
But I asked around and a ticket will be created about this as this seems enotices2 related but also SDK data related via the NTDs.

For esenders implementing their own forms and reading this:

The "displayType" "SECTION" of a group is used by enotices2 to create the sections in their interface (on the left side of the screen I think).
You can use it for a similar purpose in your own applications.

For the creation of the form use the "contentType" which can be "group" or "field" and conside the displayType depending on that.

**Reply by luispozoWS** [reply]
Hello, 

Thanks for your answer
We are using our own application
For notice 5 GR-Part-DPS displayType is SECTION.
For notice 13 GR-ChangedSectionIdentifiers displayType is SECTION.  But in notice 6 for the same field the displayType is GROUP.

Why do we have this for these two notices ? It shouldn't be the same as in the other 38 notices ?

Best regards,

**Reply by rouschr** [reply, marked as answer]
Hello, 

what I know is that it will be fixed in SDK 1.11.0, for older versions please contact enotices2 support:
https://enotices2.ted.europa.eu/help/contact
If you use your own application you should rely on the "contentType" to decide what is a group or a field.
You can try to use the displayType after you read the contentType. For the problematic cases you can either fix the SDK locally on your side or add extra code to fix this in your application until the SDK 1.11 comes out.
